pkgs/data/fonts/ultimate-oldschool-pc-font-pack/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "ultimate-oldschool-pc-font-pack-${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "http://int10h.org/oldschool-pc-fonts/download/ultimate_oldschool_pc_font_pack_v${version}.zip";
+    sha256 = "7666cf23176e34ea03a218b5c1500f4ad729d97150ab7bdb7cf2adf4c99a9a7a";
+  };
+
+  buildInputs = [ unzip ];
+
+  dontBuild = true;
+
+  sourceRoot = ".";
```

Comment:
It could be much simpler using ```fetchzip``` instead of ```stdenv.mkDerivation```

For example: https://github.com/NixOS/nixpkgs/pull/46198/files

Reply:
Resolved in #47313